Sources/ArcGISToolkit/Components/Form/TextInputFooter.swift

```diff
@@ -74,6 +76,7 @@ struct TextInputFooter: View {
         default:
             fatalError("TextInputFooter can only be used with TextAreaFormInput or TextBoxFormInput")
```

Comment:
```suggestion
            fatalError("\(Self.self) can only be used with \(TextAreaFormInput.self) or \(TextBoxFormInput.self)")
```

This might be more resilient if the type name changes but this string is missed.